packages/stylelint/source/plugins/order.ts

```diff
@@ -0,0 +1,17 @@
+import type { Config } from "stylelint";
+
+// https://github.com/hudochenkov/stylelint-order
+const config: Partial<Config> = {
+	plugins: ["stylelint-order"],
+
+	rules: {
+		"order/order": ["custom-properties", "declarations"],
+		"order/properties-order": [
```

Comment:
@acwo 

There you can continue with what you already have configured before for the company.
Let me know if you encounter any issues.